### PR TITLE
Bug 1862903: "Add disk" button exist in the wizard but not active

### PR DESCRIPTION
**Pull request by glekner**

### CODE REVIEW

**Comment by glekner** [comment]
@yaacov @suomiy please review

File: frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/tabs/networking-tab/networking-tab.tsx [hunk 1]
```diff
@@ -127,9 +127,11 @@ const NetworkingTabComponent: React.FC<NetworkingTabComponentProps> = ({
             <Title headingLevel="h5" size="lg">
               No network interface added
             </Title>
-            <Button {...addButtonProps} icon={<PlusCircleIcon />} variant={ButtonVariant.link}>
-              {ADD_NETWORK_INTERFACE}
-            </Button>
+            {!isCreateDisabled && (
```
Comment (suomiy) [inline frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/tabs/networking-tab/networking-tab.tsx:130]: we should include the buttons itself too (not just the empty flow)

Comment (glekner) [reply frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/tabs/networking-tab/networking-tab.tsx:130]: added

**Pull request merged**